charts/gardener-extension-shoot-lakom-admission/charts/runtime/templates/rbac.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "name" . }}
+  namespace: garden
```

Comment:
This chart has been reworked from the chart that is used for the `registry-cache`  extension, so I'm not completely sure why the role is for the garden namespace. 

I don't see any self-bootstrapping functionality on the admission controller, so it might be something specific for the `registry-cache` that can be safely removed for the lakom admission controller. Another possibility is that the `registry-cache` also borrowed this chart from some other extension. I'll ask them and get back to you

Reply:
@vpnachev I'll be resolving since it seem like we'll have to talk with Tim Usner about this. Based on the convo with Ismail, it's best we leave the namespace as `garden` for now.